model/database/spans.yaml

```
@@ -102,6 +102,9 @@ groups:
         sampling_relevant: true
         requirement_level:
           conditionally_required: If available.
+      - ref: db.stored_procedure.name
+        requirement_level:
+          recommended: If operation represents a stored procedure execution.
```

same here: do we want to limit it to the execution? maybe

```suggestion
          recommended: If operation applies to a specific stored procedure.
```